components/HomeScreen/HomeScreen.jsx

```diff
@@ -0,0 +1,11 @@
+import { Text, SafeAreaView } from "react-native";
+
+const HomeScreen = () => {
```

Comment:
Bit of code golf, but we've generally chosen to use `function HomeScreen() {` instead of using an arrow function. Pretty much everything in [this article](https://www.makeuseof.com/javascript-arrow-functions-careful/) are the reasons, although we do typically return an arrow function when a function is returning another function because of clarity